nexus/src/external_api/http_entrypoints.rs

```diff
@@ -3074,25 +3067,24 @@ async fn instance_stop(
 async fn instance_serial_console_v1(
     rqctx: Arc<RequestContext<Arc<ServerContext>>>,
     path_params: Path<params::InstancePath>,
-    query_params: Query<params::InstanceSerialConsole>,
+    query_params: Query<params::InstanceSerialConsoleRequest>,
+    selector_params: Query<params::OptionalProjectSelector>,
```

Comment:
So, this is a bit weird. It's implemented on the same principle as described in the docs here: https://docs.rs/dropshot/latest/dropshot/#advanced-usage-notes. Even though this endpoint doesn't do pagination, we can't flatten the `InstanceSerialConsoleRequest` and `ProjectSelector` structs into a single struct due to https://github.com/nox/serde_urlencoded/issues/33.